File: test_temporada.py
temporadaRaw = "T. ACTUAL"
actuales = ['2026-Q3', '2026-Q4', '2027-Q1', '2027-Q2', 'ACTUAL']
temporadaClean = 'ANTERIOR'
for act in actuales:
    if act in temporadaRaw:
        temporadaClean = 'ACTUAL'
print(temporadaClean)

File: test_temporada.js
const skuTemporada = { '6646806': 'T. ACTUAL' };
let temporadaRaw = skuTemporada['6646806'] || 'DESCONOCIDA';
let temporadaClean = 'ANTERIOR'; 
const actuales = ['2026-Q3', '2026-Q4', '2027-Q1', '2027-Q2', 'ACTUAL'];
if (actuales.some(act => temporadaRaw.includes(act))) {
    temporadaClean = 'ACTUAL';
}
console.log(temporadaClean);
